### ISSUE TITLE
SDR 3 11.0

### ISSUE THREAD

**Issue by KP4AKB** [ExpertSDR3/ExpertSDR3-BUG-TRACKER#195]
Installed 11.0 over 9.0 .   9.0 was removed and 11 installed without a problem. 
I get an IP and 11 starts but its dead. No signal, no audio
![image](https://user-images.githubusercontent.com/96546367/147667087-8bacaa1c-6074-42f3-9d53-23d2243c73ef.png)

Ed Rios
Columbus, OH

**Comment by G4ENZ-Martin** [comment]
Ed,

When I first started testing 0.11.0, I deleted my previous configuration and built everything up from scratch again.  You might just want to let ESDR3 make a new default configuration folder for you and see if that helps.  It should come up on 1950kHz AM making a lot of noise!

73 Martin

**Comment by KP4AKB** [comment]
Martin, I will try your suggestion

**Comment by KP4AKB** [comment]
Martin, I uninstalled SDR 11 and deleted the folder, Reinstalled it again and some functions work, but there is no signal.
I went back to SDR 9.0 and it works as before. Does the firmware make a difference? 

I am at a loss.

Ed Rios 

    On Wednesday, December 29, 2021, 11:05:52 AM EST, G4ENZ-Martin ***@***.***> wrote:  
 
 


Ed,

When I first started testing 0.11.0, I deleted my previous configuration and built everything up from scratch again. You might just want to let ESDR3 make a new default configuration folder for you and see if that helps. It should come up on 1950kHz AM making a lot of noise!

73 Martin

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by G4ENZ-Martin** [comment]
Hi Ed,

I am running Firmware 72.8 on SunSDR2 Pro but I do not think there has been a change to the my firmware since ESDR2 update 11 which was October 2021. I see you are running Firmware 7.3 which, I guess, is specific to the ColibriDDC. I have no knowledge of what the "proper" version is for your radio.

Sorry,
Martin

**Comment by ExpertSDR3** [comment]
If there are no signals on the panorama, most likely the firewall blocks the programs.

**Comment by KP4AKB** [comment]
I found the answer, You must have the latest firmware installed on ColibriDDC. I had firmware 7.3 because the S meter works. 
When I upgraded to 11.3 alpha I installed latest firmware  11.3 and all works now. But the s mater reads too high by 35dB

Ed RiosKP4AKB/W8Columbus, OH
 

    On Tuesday, January 18, 2022, 02:24:21 AM EST, EE ***@***.***> wrote:  
 
 


If there are no signals on the panorama, most likely the firewall blocks the programs.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
